ament_cmake_flake8/cmake/ament_flake8.cmake

```
@@ -57,7 +59,9 @@ function(ament_flake8)
   if(ARG_MAX_LINE_LENGTH)
     list(APPEND cmd "--linelength" "${ARG_MAX_LINE_LENGTH}")
   endif()
-  list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
+  if(ARG_EXCLUDE)
+    list(APPEND cmd "--exclude" "${ARG_EXCLUDE}")
+  endif()
```

You still want to add `ARG_UNPARSED_ARGUMENTS` to your command once you're done processing all arguments.


```suggestion
  if(ARG_EXCLUDE)
    list(APPEND cmd "--exclude" "${ARG_EXCLUDE}")
  endif()
  
  list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
```